tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,21 @@
+k = 1
+
+
+def get_digits_number(num: int) -> int:
+    count = 0
+    while num > 0:
+        count += 1
+        num = num // 10
+    return count
+
+
+def is_narcissistic_num(num: int) -> bool:
+    n: int = get_digits_number(num)
+    nth: [int] = [int(digit) ** n for digit in str(num)]

```

Comment:
The string solution is good.
Please check 
https://docs.google.com/document/d/1EgIJw4_dZehaOURjYDdtZI09jMlQe2AGmfneZUgxfDM/edit#
There is a note in a task to use pure math solution.

Reply:
I think an additional note was added recently ?